packages/applications-service-api/src/utils/advice.mapper.js

```diff
@@ -0,0 +1,26 @@
+const mapBackOfficeAdviceToApi = (backOfficeAdvice) => {
+	return backOfficeAdvice.map((advice) => {
+		const [firstName, ...lastName] = advice && advice.from ? advice.from.split(' ') : undefined;
+		return {
+			s51Enquiry: true,
```

Comment:
I think this should be named `section51Enquiry`, as per existing API / NI db

https://github.com/Planning-Inspectorate/applications-service/blob/3f3b942c396129dfcbe884807b13c5f44d48cdd2/packages/applications-service-api/src/models/advice.js#L24